supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -86,32 +86,16 @@ object TesAsyncBackendJobExecutionActor {
        |  command -v "$$1" > /dev/null 2>&1
        |}
        |
-       |# Check if curl exists; install if not
+       |# Require the user image to pre-install `jq` and `curl` for us. Empirically, we found
+       |# that `apt install` at scale can run into repo outages and flakiness.
        |if ! command_exists curl; then
-       |  if command_exists apt-get; then
-       |    apt-get -y update && apt-get -y install curl
-       |    if [ $$? -ne 0 ]; then
-       |      echo "Error: Failed to install curl via apt-get."
-       |      exit 1
-       |    fi
-       |  else
-       |    echo "Error: apt-get is not available, and curl is not installed."
-       |    exit 1
-       |  fi
+       |  echo "Error: user image must include `jq` for just-in-time SAS token generation, but it is not installed."
```

Comment:
I think these two error messages need to be reversed.

Reply:
🤦‍♂️ thank you!